students/sangwoong/users/views.py

```diff
@@ -41,4 +43,28 @@ def post(self, request) :
             return JsonResponse({"message": "SUCCESS"}, status=201)
 
         except:
-            return JsonResponse({"message": "KEY_ERROR"}, status=400)
\ No newline at end of file
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            user_account  = data["account"]
+            user_password = data["password"]
+
+            is_account_blank  = re.match("^$", user_account)
+            is_password_blank = re.match("^$", user_password)
+
+            if is_account_blank or is_password_blank :
+                return JsonResponse({"Message": "KEY_ERROR"}, status=400)
+
+            if not User.objects.filter(account = user_account).exists() :
+                return JsonResponse({"Message": "INVALIDE_USER"}, status=401)
+
+            if User.objects.get(account = user_account).password != user_password :
+                return JsonResponse({"Message": "CHECK_YOUR_ACCOUNT_OR_PASSWORD"}, status=401)
```

Comment:
로그인의 경우 아이디가 잘못 되었는지, 비밀번호가 잘못 되었는지를 모르게 하는 것이 보안에 더 좋습니다.

```suggestion
            if User.objects.get(account = user_account).password != user_password :
                return JsonResponse({"Message": "INVALIDE_USER"}, status=401)
```